plugin/forward/setup.go

```diff
@@ -92,7 +92,7 @@ func parseStanza(c *caddy.Controller) (*Forward, error) {
 	if !c.Args(&f.from) {
 		return f, c.ArgErr()
 	}
-	f.from = plugin.Host(f.from).Normalize()
+	f.from = plugin.Host(f.from).Normalize()[0] // there can only be one here, won't work with non-octet reverse
```

Comment:
Should be in README.

Reply:
Until it is supported, we should probably error out if `from` Normalizes to > 1 zone.

Reply:
Opened #4639 to address this in README and as a start up _warning_.